.evergreen/config.yml

```diff
@@ -3259,6 +3249,99 @@ buildvariants:
     AUTH: auth
     PYTHON_BINARY: /opt/python/pypy3.10/bin/python3
 
+# PyOpenSSL tests.
+- name: pyopenssl-macos-py3.9
+  tasks:
+    - name: .replica_set
+    - name: .7.0
+  display_name: PyOpenSSL macOS py3.9
+  run_on:
+    - macos-14
+  batchtime: 10080
+  expansions:
+    AUTH: noauth
```

Comment:
Should this have auth as well? Every other task here does.

Reply:
No, there's a note in the generator that is carried over from the config.yml file